appveyor.yml

```diff
@@ -52,6 +52,8 @@ after_build:
 - cmd: C:\projects\spectre\scripts\package_project.bat DivikResultConverter Spectre.DivikResultConverter
 - cmd: C:\projects\spectre\scripts\package_web_project.bat WebAPI Spectre
 - cmd: C:\projects\spectre\scripts\package_angular_client.bat
+- cmd: C:\projects\spectre\scripts\build_ng_client.bat Bit
+- cmd: C:\projects\spectre\scripts\package_bit_client.bat
```

Comment:
Please move `build_ng_client.bat` command to `build_script` section above; `after_build` should have only packaging scripts.

Reply:
done, bit build is created in bitDist directory